app/reflexes/example_reflex.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class ExampleReflex < ApplicationReflex
```

Comment:
Why do we need this example file? If you want the documentation then we could add that to the ApplicationReflex, maybe?

Reply:
I usually have this around so that one can just copy this file when creating a new reflex file. Just like how we have an `example_controller.js` in `app/webpacker/controllers`